src/CSLibrary/CSLibrary.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-   <PackageReference Include="KubernetesClient" Version="4.0.13" />
+   <PackageReference Include="KubernetesClient" Version="7.0.6" />

```

Comment:
Any reason to not go to 7.2? Looks like that's the latest stable version according to the [version chart](https://github.com/kubernetes-client/csharp).